content/en/software_catalog/eng_reports/reliability_overview.md

```diff
@@ -0,0 +1,107 @@
+
+You can update your Reliability Overview report view in the following ways:
+ 
+- **Switch the aggregation between "Service" or "Team"**: View your organization's SLO and Incident performance by service/team groupings to identify top- and bottom-performing areas. 
+
+   **Note**: Service/team groupings are based on the **service** or **team** tag added to your SLOs, and the **services** or **teams** properties added to your Incidents.
+
+- **View "Daily", "Weekly", or "Monthly" historical information**: Update the historical SLO and Incident trends to your desired granularity.
+- **Add filters to scope the data**: Filter by teams, services, and incident severity and state.
+
+### Schedule reports
+
+Set up scheduled reports for your stakeholders that will be delivered as PDFs through Email or Slack on a recurring basis.
+
+To schedule reports, click on **Schedule Report** in the top right corner (or **Manage Reports** if you've already set up reports). Refer to the [Scheduled Reports documentation][1] for more information.
+
+### Customize your report 
+
+On the upper right corner of the report, click the kebab menu and select **Clone as a Dashboard** to create a dashboard with content from the Reliability Overview report. 
+
+To customize the dashboard, you can:
+- Update the SLO Summary table to group by any tag you have added to your SLOs (for example, you can create a view grouped by "user journey")
+- Add widgets that are not included in the default view
+- Add filters to the existing widgets (for example, you can filter Incidents based on "Detection Method" not provided in the out-of-the-box report)
+
+## Use the SLO summary score
+
+{{< img src="tracing/eng_reports/slo-summary-score2.png" alt="The SLO Summary widget, including the SLO summary score" style="width:100%;" >}}
+
+The **SLO Summary** widget includes a "Score". It is designed as a summary metric for executive leadership to understand the performance of a group of SLOs. The Score is calculated based on the average remaining error budget of the underlying SLOs, which is then mapped to a score between 0 - 100:
+
+- The Score is "passing" (green/yellow) when most SLOs are **not** breached and have remaining error budget
+- The Score is "failing" (red) when many SLOs are out of error budget or a few SLOs are far out of error budget
+- SLOs in the "No Data" state are not considered in the Score
+
+### Score calculation details
+
+The Score is calculated as follows:
+
+{{< jqmath-vanilla >}}
+
+$$
+\text"Average Remaining Error Budget"
+      = {∑_{i=0}^{n}\\text"[Remaining Error Budget]"_i} / n
+$$
+
+$$
+\text"Score"
+      = {max(\text"[Average Remaining Error Budget]"\,-200) + 200} / 300 * 100
+$$
+
+
+- Average the remaining error budget of the SLOs (the minimum error budget is set to -200%, so any SLO with a lower error budget will be counted as -200% in the average)
+- The average error budget (between -200 and 100) is mapped to a Score between 0 and 100
```

Comment:
```suggestion
- The average error budget (between -200 and 100) is mapped to a Score between 0 and 100.
```
Might be worth giving each of these bullets the same structure, like "Map the average to ..."